StandardSocketsHttpHandler/Net/Http/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -13,9 +13,9 @@ public static bool HasHeaders(this HttpRequestMessage request)
             bool isDotNetFrameworkOrMono = isDotNetFramework || RuntimeUtils.IsMono();

```

Comment:
So we don't need the 'isDotNetFramework' variable anymore...

Reply:
Indeed. And to avoid repetitive costly reflection calls, I'll cache the fieldinfo into a static field.

Reply:
And I will drop the framework dependant fallback: I'll just try with "headers" then "_headers"